### ISSUE TITLE
Vulnerabilidade

### ISSUE THREAD

**Issue by Doges** [saulojoab/Syntoma#1]
Vulnerabilidade grave por conta da exposição do GET do json:
https://github.com/saulojoab/Syntoma/blob/75a1c4340d3925322ca210af8164ce812bea48c7/src/cadastro.js#L9

**Comment by saulojoab** [comment]
Sim, eu sei. Como foi pouco tempo pra fazer o app pro concurso, eu fiz às pressas e não adicionei praticamente nenhuma camada de segurança. Em releases futuros isso será corrigido. 

Cheers 👍
